.github/workflows/label_pull_request_onCreate.yml

```diff
@@ -0,0 +1,21 @@
+# Workflow that labels a pr with the correct label
+name: "[Github Management] Pull Request Labeler on create or taken out of draft"
+
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+
+jobs:
+  needs_review:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Add FE-Needs Reviewv
+        run: |
+          gh pr edit ${{ github.event.pull_request.number }} --add-label "FE-Needs Review"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We'll probably want logic to prevent this from running on non FE related PRs (e.g. documentation PRs where other teams are updating their own docs). You can make this run conditionally based on whether `flagship-mobile-reviewers` is requested

Reply:
I personally disagree. I know I and a few others in the FE have looked at and approved documentation PR's for other teams and it doesn't affect them as they didn't use the label anyways. How exactly would I add that if check though if you still disagree?

Reply:
The main purpose of the `FE-Needs Review` label is to make it easier to filter which PRs need to be reviewed by FE. Based on the current logic, every PR created in the repo would have this label, and while most PRs do need our review, not all do (e.g. PRs with changed files that have other teams as the sole codeowner). If other teams update documentation that need our review, we'd be marked as a reviewer on the PR, and the label would still be added, even with my suggested logic changes. 

You can add that check with
``` yaml
if: ${{ github.event.requested_team.name == 'flagship-mobile-reviewers'}}
```

Reply:
added